site/docs/es/plugins/README.md

```diff
@@ -21,6 +21,40 @@ Cada sección de la documentación de un plugin oficial tiene el nombre del paqu
+| ---------------------------------- | ----------------------------------------------------- | --------------------------------------------------------------------------- |
+| Sesiones                           | _incluido_                                            | Almacena los datos de los usuarios en su base de datos                      |
+| Teclados en línea y personalizados | _incluido_                                            | Simplifica la construcción de teclados en línea y personalizados            |
+| Auto-reintento                     | [`auto-retry`](./auto-retry.md)                       | Gestiona automáticamente la limitación de la velocidad                      |
+| Conversaciones                     | [`conversations`](./conversations.md)                 | Construir potentes interfaces y diálogos conversacionales                   |
+| Emojis                             | [`emoji`](./emoji.md)                                 | Simplificar el uso de emoji en el código                                    |
+| Archivos                           | [`files`](./files.md)                                 | Fácil manejo de archivos                                                    |
+| Hidratar                           | [`hydrate`](./hydrate.md)                             | Llamar a los métodos de los objetos devueltos por las llamadas a la API     |
+| Internacionalización               | [`i18n`](./i18n.md) or [`fluent`](./fluent.md)        | Deja que tu bot hable varios idiomas                                        |
+| Menús interactivos                 | [`menu`](./menu.md)                                   | Diseñe menús de botones dinámicos con navegación flexible                   |
+| Parsear                            | [`parse-mode`](./parse-mode.md)                       | Simplificar el formato de los mensajes                                      |
+| Limitador de velocidad             | [`ratelimiter`](./ratelimiter.md)                     | Restringe automáticamente a los usuarios que envían spam a tu bot           |
+| Enrutador                          | [`router`](./router.md)                               | Dirija los mensajes a diferentes partes de su código                        |
+| Concurrencia                       | [`runner`](./runner.md)                               | Long polling csimultáneo a escala                                           |
+| Preguntas sin estado               | [`stateless-question`](./stateless-question.md)       | Crear diálogos sin almacenamiento de datos                                  |
+| Control de flujo                   | [`transformer-throttler`](./transformer-throttler.md) | Poner en cola automáticamente las llamadas a la API para evitar las esperas |
+
+También tenemos algunos plugins de terceros.
+Puedes encontrarlos en el menú de navegación en _Plugins_ > _Third-Party_.
```

Comment:
> _Plugins_ > _Third-Party_

@habemuscode is this navigation menu correct? I think the correct one for ES page is `_Complementos_ > _De Terceros_`.

Reply:
Will be fixed in https://github.com/grammyjs/website/pull/558

Thanks